app/controllers/idv/hybrid_handoff_controller.rb

```diff
@@ -47,9 +47,11 @@ def self.selected_remote(idv_session:)
       if IdentityConfig.store.in_person_proofing_opt_in_enabled &&
```

Comment:
Hey @KeithNava, I am confused about what’s being displayed in “Files changed” section of this PR.  When I look at the code on this branch locally, I see your changes to `HowToVerifyController`.  But the `HowToVerifyController` doesn’t show up in the “Files changed” section of this PR.  

However, if I navigate to the `keithw/LG-13006-rename-skip-doc-auth-value` branch and look at the `HowToVerifyController`, I see your commit [58d401d](https://github.com/18F/identity-idp/commit/58d401d4515c8897dec62e6f05bfae34d613d0e2)  from four days ago.  Link [here](https://github.com/18F/identity-idp/blob/keithw/LG-13006-rename-skip-doc-auth-value/app/controllers/idv/how_to_verify_controller.rb). 

Any ideas what might have caused this?  It has me scratching my head. 🤔

Reply:
@eileen-nava  I'm not exactly sure what happening there but I created a few commits around whitespacing to try and 'trick' git into not noticing a diff.